<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConcatDialog</class>
 <widget class="QDialog" name="ConcatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider::groove:horizontal{
background:#000000;
height:8px;
border-radius:4px;
}

QSlider::add-page:horizontal{
background:#E4E4E4;
height:8px;
border-radius:4px;
}

QSlider::sub-page:horizontal{
background:#000000;
height:8px;
border-radius:4px;
}

QSlider::handle:horizontal{
width:13px;
margin-top:-3px;
margin-bottom:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #FFFFFF,stop:0.8 #B6B6B6);
}
QPushButton {
background-color:#ffffff;
border-width:2px;
border-color: #000000;
border-style: solid;
border-radius:10px;  
}
QGroupBox{
    border:2px solid;
    border-radius:5px;
    margin-top:0.5em;
}

QGroupBox::title{
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding:0 1px;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_show1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_show2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_show">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>371</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>141</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>简单拼接</string>
   </property>
   <widget class="QPushButton" name="pushButton_fisheye">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>鱼眼拼接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_horizomal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>水平方向拼接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vertical">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>垂直方向拼接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bend">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>平面曲翘拼接</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>320</y>
     <width>151</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>匹配拼接</string>
   </property>
   <widget class="QPushButton" name="pushButton_SURF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SURF特征匹配拼接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>基于模板匹配拼接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SIFT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT特征匹配拼接</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_open1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>250</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图像</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConcatDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConcatDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
